docs/proposals.md

```diff
@@ -0,0 +1,29 @@
+libnetwork proposals
+====================
+
+This is a working document for proposals that will be made to Docker Engine.
+
+## Replace Networking Subsystem with libnetwork
+
+< todo >
+
+## Add "Network" as a first class object in Docker
+
+### CLI Changes
+
+
+| Command | Flags | Arguments | Example |
+|---------|-------|---------|---------|
+| docker network create | -d, --driver <br> -n, --name | | docker network create -d overlay --name=foo |
```

Comment:
We need an option to pass config/labels in the cli/api that libnetwork can handover to the drivers.

Example:
docker network create --driver myDriver --name myNet --label prop1=val1 --label prop2=val2